config/localization/en/app.json

```diff
@@ -1230,6 +1233,8 @@
       "DEPOSIT_SUCCESS": "Deposit success!",
       "DIRECT_TRANSFER_SUCCESS": "Direct transfer success!",
       "DIRECT_TRANSFER_SUCCESS_DESCRIPTION": "Your direct transfer of {AMOUNT_ELEMENT} has been confirmed.",
+      "DISABLE_PUSH_NOTIFICATIONS": "Disable Push Notifications",
+      "ENABLE_PUSH_NOTIFICATIONS": "Enable Push Notifications",
```

Comment:
should this be "Disable push notifications" vs "Disable Push Notifications"?

Reply:
I think capitalized should be okay. Can change in the future.